projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -251,6 +251,13 @@ export interface FeatureTogglesInterface {
    * `LoginRegisterComponent`, `ConfigureProductComponent`
    */
   a11yUseButtonsForBtnLinks?: boolean;
+
+  /**
+   * When enabled disable "Apply" button in promo code component in cart for empty input,
+   * disable "Add" button in quick order component when input is empty and remove
+   * required validators for both inputs
+   */
+  a11yDisabledButtonInsteadOfRequiredFields?: boolean;
```

Comment:
This name could describe which buttons are being disabled to distinguish it a bit more.